### ISSUE TITLE
List dependencies for Ubuntu / apt

### ISSUE THREAD

**Issue by alexellis** [projectatomic/buildah#243]
Please could you list the dependencies for an Ubuntu / apt-based repo in the readme?

**Comment by rhatdan** [comment]
@alexellis Are you talking about the stuff we have to build buildah?
Travis has
```
    - sudo add-apt-repository -y ppa:duggan/bats
    - sudo apt-get -qq update
    - sudo apt-get -qq install bats btrfs-tools git libapparmor-dev libdevmapper-dev libglib2.0-dev libgpgme11-dev libselinux1-dev
    - sudo apt-get -qq remove libseccomp2
```

**Comment by alexellis** [comment]
Yes - the bootstrap, so yum is mentioned on the README front page, but coming from a background working mainly with Ubuntu it might help interest if it was updated to show instructions for both.

**Comment by alexellis** [comment]
This is obviously missing the point of the project - but it might also be worth adding a Dockerfile that documents the complete build - is repeatable and testable and can be run easily by MacOS users to evaluate the project without having to install a Linux VM.

**Comment by rhatdan** [comment]
Open a pull request for what you would like to see.  Also please review 244, to tell me if it works.

**Comment by alexellis** [comment]
Thanks! It'll be a few days if you can wait.

**Comment by anarcat** [comment]
i think this should probably be closed, considering #244 is merged and #354 is pending.

**Comment by nalind** [comment]
Makes sense to me.  Closing.
